Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -172,7 +172,27 @@ public final class SecureEnclaveValet: NSObject {
             try SecureEnclave.string(forKey: key, withPrompt: userPrompt, options: baseKeychainQuery)
         }
     }
-    
+
+    /// Attempts to retrieve the string at the given key, allowing a custom `fallbackTitle` to be specified to be
+    /// shown to the user in the case that the biometric authentication fails.
+    ///
+    /// - Parameters:
+    ///   - key: A key used to retrieve the desired object from the keychain.
+    ///   - userPrompt: The prompt displayed to the user in Apple's Face ID, Touch ID, or passcode entry UI.
+    ///   - fallbackTitle:  The title of the fallback button shown to the user after 2 failed retrieval attempts.
+    ///                     If the user taps this button, an `SecureEnclaveError.userFallback` will be thrown.
+    /// - Returns: The string currently stored in the keychain for the provided key.
+    /// - Throws: An error of type `KeychainError` or `SecureEnclaveError`.
+    public func string(
```

Comment:
Any reason not to expose this method to Objective-C? I think the way we've written it all we need is an `@objc` tag.

Reply:
Nope! Can do 👍